lib/cisco_node_utils/cmd_ref/ospf_area.yaml

```diff
@@ -38,6 +38,35 @@ filter_list_out:
   set_value: '<state> area <area> filter-list route-map <route_map> out'
   default_value: false
 
+nssa:
+  multiple: true
+  get_value: '/^area <area> nssa(.*)$/'
+  set_value: '<state> area <area> nssa <no_summary> <no_redistribution> <default_information_originate> <route_map> <rm>'
+
+nssa_def_info_originate:
+  kind: boolean
+  default_value: false
+
+nssa_enable:
+  kind: boolean
+  default_value: false
+
+nssa_no_redistribution:
```

Comment:
I dislike using 'no' properties that disable behavior when they're set to true. It's confusing just to write that.

In other words, I prefer:

cli | better | worse
:--|:--|:--
`nssa no-redistribute` | `redistribution => false` |  `no_redistribution => true`

...but this is a personal preference. I'd like to get some feedback from Mike and Rob as well.